cache.go

```diff
@@ -217,21 +217,25 @@ func (a *Cache[K, V]) Clear() {
 	a.policy.Clear()
 }
 
-// Noop if smaller. available should not consider taken space in cache.
+// Noop if smaller. available (+/-) should not consider taken space in cache.
 func (a *Cache[K, V]) SetLargerCapacity(available, max int64) {
+	a.setCapacity(available, max, true)
+}
+
+// available (+/-) should not consider taken space in cache.
+func (a *Cache[K, V]) SetCapacity(available, max int64) {
+	a.setCapacity(available, max, false)
+}
+
+func (a *Cache[K, V]) setCapacity(available, max int64, largerOnly bool) {
 	for {
 		cap := a.cap.Load()
-		size := a.size.Load()
```

Comment:
taking size out of the equation, since there is no more "growing" in the implementation seems appropriate.